reviews two/swagger.yaml

```diff
@@ -0,0 +1,879 @@
+        - api_key: []
+            
+      x-code-samples:
+        - lang: curl
+          source: curl https://myshopify.com/v1/admin/draft_orders.json
+      
+        - lang: Ruby
+          source: |
+              # load the gem
+              require 'swagger_client'
+              
+              api_instance = SwaggerClient::GetApi.new
+              
+              opts = { 
+                fields: "fields_example", # String | Comma-separated list of fields to include in the response
+                limit: 789, # Integer | Page size
+                page: 789 # Integer | Page to show
+              }
+              
+              begin
+                result = api_instance.admin_draft_orders_json_get(opts)
+                p result
+              rescue SwaggerClient::ApiError => e
+                puts "Exception when calling GetApi->admin_draft_orders_json_get: #{e}"
+              end
+            
+            
+        - lang: PHP
+          source: |
+            <?php
+            require_once(__DIR__ . '/vendor/autoload.php');
+            
+            $api_instance = new Swagger\Client\Api\GetApi();
+            $fields = "fields_example"; // string | Comma-separated list of fields to include in the response
+            $limit = 789; // int | Page size
+            $page = 789; // int | Page to show
+            
+            try {
+                $result = $api_instance->adminDraftOrdersJsonGet($fields, $limit, $page);
+                print_r($result);
+            } catch (Exception $e) {
+                echo 'Exception when calling GetApi->adminDraftOrdersJsonGet: ', $e->getMessage(), PHP_EOL;
+            }
+            ?>
+                  
+        - lang: Python
+          source: |
+            
+            # create an instance of the API class
+            api_instance = Shopify.GetApi()
+            fields = 'fields_example' # str | Comma-separated list of fields to include in the response (optional)
+            limit = 789 # int | Page size (optional)
+            page = 789 # int | Page to show (optional)
+            
+            try: 
+                api_response = api_instance.admin_draft_orders_json_get(fields=fields, limit=limit, page=page)
+                pprint(api_response)
+            except ApiException as e:
+                print "Exception when calling GetApi->admin_draft_orders_json_get: %s\n" % e
+
+ 
+
+
+          
+            
+  
+    post:
+      summary: Create a draft order
+      description: >
+        Create a draft order
+          ### About line items
+          There are two ways to add a line item:
+
+          * Supply a `variant_id`, `quantity` and (if applicable) `discount`
+          * Add a custom line item by supplying `title`, `price`, `taxable` and `quantity` (at a minimum). The array of line items on a draft order must not contain a `variant_id` more than once. The data copied over from a variant upon creation of a line item is never subsequently updated.
```

Comment:
This text should be on the overall line item:
- The array of line items on a draft order must not contain a `variant_id` more than once.

And this text should be on the line above since it talks about the variant line items, not custom line items:
- The data copied over from a variant upon creation of a line item is never subsequently updated.